### ISSUE TITLE
Screen Reader Accessibility regression for Blind users

### ISSUE THREAD

**Issue by valiant8086** [lich426/FanCtrl#62]
I'm blind and have been tinkering with this program for a while now, starting somewhere around the 1.4 version.

As a blind user, I discovered that the other fan control applications are generally inaccessible with screen readers. They're just too graphical and there's nothing for the screen reading application like NVDA or JAWS to read. If it does read anything, it's too discombobulated for a user to actually work with, being unable to understand which control is associated with what.

On the other hand though, I found that Fan Ctrl, which I had originally confused for Fan Control which is not accessible, is actually reasonably accessible. I can tab around, and things read well enough that I can figure out what I'm adjusting. I can arrow up and down in the spin boxes for my fans and get immediate gratification as my fans adjust as commanded.

Auto Fan Control was harder to set up. I ended up being able to figure out how to copy an existing .preset file from the presets folder under the program's directory and tweak the numbers directly in the file and then load that in the audo fan control page of the program and test it, and maybe edit the file if still not exactly what I want until I get it.

Basically my point here is Fan Ctrl is uniquely useful for blind users as one of the only programs to control fan speed. I build computers as a hobby, and I find this program to be incredibly useful in getting the best bang for buck out of new cooling designs I'm investigating.

Ordinarily, a person might just run the Asus AI Suite 3 and set up fan speeds there for their new Asus based build and call it good, but since that program is not usable for screen reader users, and there is no way to have a screen reader to provide speech output for a blind user in the BIOS, nor are there obvious accessible fan control applications for MSI and AsRock. That said, Fan Ctrl is the solution and we have been very grateful for that.

I do actually have an "issue" to talk about. Version 1.5.0 is still normal for usability, but 1.5.1 is different. The change is probably going to be confusing to understand but I'll give it a try.

So when you use this with a keyboard, the tab key moves from control to control. If you have a bunch of temp sensors, and a bunch of fans that you can tab to their current speed percentages and the names thereof, and also tab through the current reading of rpm for each one, there's a whole lot of busy tabbing to do to get places. Sighted people can test this, just hit tab and try to visually watch where your focus goes while focused on the program. You can also start Narrator, the built-in screen reader in Windows by pressing ctrl+windows+enter, and then try this tabbing and arrowing you can hear the voice telling you where you are.

Note: that's a toggle so ctrl+windows+enter will turn Narrator back off again as well.

I accidentally discovered however, that if I'm focused on the auto fan control button and then I press tab, I land somewhere different than if I down arrow or up arrow from that auto fan control button to one of the other buttons and then tab while on that button. It gave us a nifty way to jump between the various portions of the program, be that the fan rpm readouts, the fan speed adjustments, or the temp sensors.

So saying, that no longer works, when I try to do this, for instance, arrowing down from the auto fan button and then I press my tab key, I just go to other buttons, so no matter what I do now I have to tab past my temp sensors to get to the adjustments I can make, or shift+tab the other way around, etc. It works better than anything else, but it does not work as easily as it did in 1.5.0 so I have not been taking advantage of the amazing improvements in newer versions.

What I'd like to request is an attempt to return to the original behavior with a keyboard, or just to add something that would give easier access to move keyboard focus between the distinctive sections. F6 is used a lot in Windows to jump between panes, like in a browser it takes you from the page to the address bar to maybe a tool bar, etc.

Perhaps the developer could be convinced to implement f6 to cycle between the temp sensors, focusing on the first one when pressed, the fan speed readouts in rpm, focusing the first one when pressed, and the fan control adjustments, focusing on the first one when pressed. F6 would cycle from one to the other and wrap back around to the first one again.

I have not looked at the auto fan control area. I know from the change log that it has been redesigned. I'll create another ticket if I have anything there. I can understand why I can't do much useful with the graphs in there or the fan curve, as that's too visual in nature I think, but if we can figure out a way to work around that without having to edit files it would be beautiful. But in the meantime, being able to edit the files to do this is really nice to have.

Sorry this has got to be your longest issue message ever lol.
